torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,317 @@
+        """Load an individual single pixel timeseries.
+
+        Args:
+            path: path to the image
+
+        Returns:
+            the image
+            ImportError if h5py is not installed
+        """
+        try:
+            import h5py  # noqa: F401
+        except ImportError:
+            raise ImportError(
+                "h5py is not installed and is required to use this dataset"
+            )
+        filename = os.path.join(path)
+        with h5py.File(filename, "r") as f:
+            array = f.get("array")[()]
+            tensor = torch.from_numpy(array).float()
+            return tensor
+
+    def _load_label(self, idx: str, dataset: str) -> str:
+        """Load the croptype label for a single pixel timeseries.
+
+        Args:
+            idx: sample index in labels.geojson
+            dataset: dataset name to query labels.geojson
+        Returns:
+            the croptype label,
+            "Some" if no croptype is defined but is_crop is true,
+            or "None" if there are no crops
+        """
+        index = int(idx)
+        row = self.labels[
+            (self.labels["properties.index"] == index)
+            & (self.labels["properties.dataset"] == dataset)
+        ]
+        row = row.to_dict(orient="records")[0]
+        label = "None"
+        if row["properties.label"]:
+            label = row["properties.label"]
+        elif row["properties.is_crop"] == 1:
+            label = "Some"
+
+        return label
+
+    def _check_integrity(self) -> bool:
+        """Checks the integrity of the dataset structure.
+
+        Returns:
+            True if the dataset directories are found, else False
+        """
+        for fileinfo in self.file_dict.values():
+            filename = fileinfo["extracted_filename"]
+            filepath = os.path.join(self.root, self.directory, filename)
+            if not os.path.exists(filepath):
+                return False
+        return True
+
+    def _download(self) -> None:
+        """Download the dataset and extract it.
+
+        Raises:
+            AssertionError: if the checksum of split.py does not match
+        """
+        if self._check_integrity():
+            print("Files already downloaded and verified")
+            return
+        features_path = os.path.join(
+            self.directory, self.file_dict["features"]["filename"]
+        )
+        download_and_extract_archive(
+            self.file_dict["features"]["url"],
+            self.root,
+            filename=features_path,
+            md5=self.file_dict["features"]["md5"] if self.checksum else None,
+        )
+
+        download_url(
+            self.file_dict["labels"]["url"],
+            self.root,
+            filename=os.path.join(self.directory, self.file_dict["labels"]["filename"]),
+            md5=self.file_dict["labels"]["md5"] if self.checksum else None,
+        )
+
+    def plot(self, sample: dict[str, object], subtitle: Optional[str] = None) -> Figure:
```

Comment:
Can you add an example plot to the PR description so we can make sure plotting works correctly? Not sure if plot even makes sense for a single-pixel dataset.